src/main/java/com/google/codeu/servlets/HomeServlet.java

```diff
@@ -0,0 +1,31 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+@WebServlet("/home")
+public class HomeServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+    throws IOException, ServletException {
+
+    UserService userService = UserServiceFactory.getUserService();
+
+    boolean isUserLoggedIn = userService.isUserLoggedIn();
+    request.setAttribute("isUserLoggedIn", isUserLoggedIn);
+
+    if (userService.isUserLoggedIn()) {
```

Comment:
You just set this function call to a variable a few lines up, so you can just use that variable here

Reply:
Oh, you're right! I just made the changes in the most recent commit.